apps/codelab/src/app/codelabs/angular/router/router.component.html

```diff
@@ -117,7 +117,7 @@ <h2 i18n>Use <b>routerLink</b> directive to provide the URL</h2>
   </div>
 
   <div *slide id="exercise-pre">
-    <h1 i18n>Router / Exercise</h1>
+    <codelab-breadcrumb i18n>Exercise</codelab-breadcrumb>
```

Comment:
Is it ok, that there is no breadcrumb on exercise page?
http://localhost:4200/angular/router/exercise

Reply:
Yes, there's so much stuff there that I'd rather not have it. 
Maybe we can find some way to have it half hidden or something later?